CC/Sounder/ClientRadioSetUHD.cc

```diff
@@ -0,0 +1,236 @@
+/*
+ Copyright (c) 2018-2019, Rice University
+ RENEW OPEN SOURCE LICENSE: http://renew-wireless.org/license
+
+----------------------------------------------------------
+ Initializes and Configures Client Radios
+----------------------------------------------------------
+*/
+#include "include/ClientRadioSetUHD.h"
+
+#include "include/RadioUHD.h"
+#include "include/comms-lib.h"
+#include "include/logger.h"
+#include "include/macros.h"
+#include "include/utils.h"
+
+static void freeRadios(RadioUHD*& radios) { delete radios; }
```

Comment:
*& ... maybe there is a better way then passing a reference to a ptr for this functions

Reply:
That's true, it is kind of not pretty.